requirements-freqai-rl.txt

```diff
@@ -2,7 +2,7 @@
 -r requirements-freqai.txt
 
 # Required for freqai-rl
-torch==1.13.1; python_version < '3.11'
+torch==2.0.0; python_version < '3.11'
```

Comment:
wasn't torch 2.0 supposed to be adding support for 3.11? 

Might be worth a try if it installs and works properly on 3.11 (basically removing the constraint in the requirements file)?

Reply:
I use python 3.11 and pytorch 2.0.0, and pandas 2.0.0 in my latest test setups. It indeed works properly, but I have not installed normal Freqai requirements like xgboost and catboost etc, because I'm into RL, but python 3.11 and pytorch 2.0.0 works perfectly. :)

Reply:
I also have tested with 3.11 + pytorch 2.0.0. Did not notice any issues.

I removed the constraint from the requirements.txt.